app/config/AppConfig.scala

```diff
@@ -109,6 +109,14 @@ class AppConfig @Inject()(servicesConfig: ServicesConfig, configuration: Configu
   private def userAllowListService: String = servicesConfig.baseUrl("user-allow-list")
 
   def userAllowListBaseUrl: String = s"$userAllowListService/user-allow-list"
-}
 
+  private def traderKnownFactsReferenceDataService: String =
+    if (isEnabled(StubGetTraderKnownFacts)) {
+      servicesConfig.baseUrl("emcs-tfe-reference-data-stub")
+    }
+    else {
+      servicesConfig.baseUrl("emcs-tfe-reference-data")
+    }
 
+  def traderKnownFactsReferenceDataBaseUrl: String = s"$traderKnownFactsReferenceDataService/emcs-tfe-reference-data"
```

Comment:
Could you add a test in AppConfigSpec? Similar to how you did it [here](https://github.com/hmrc/emcs-tfe-cancel-movement-frontend/blob/bba0881290635eb3ee5d54976089bc20727e9de1/test/config/AppConfigSpec.scala#L108-L126)